pkg/controller/appmetricsservice/util.go

```diff
@@ -0,0 +1,37 @@
+package appmetricsservice
+
+import (
+	"fmt"
+	"strings"
+
+	metricsv1alpha1 "github.com/aerogear/app-metrics-operator/pkg/apis/metrics/v1alpha1"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func labels(cr *metricsv1alpha1.AppMetricsService, suffix string) map[string]string {
+	return map[string]string{
+		"app":       cr.Name,
+		"service":   fmt.Sprintf("%s-%s", cr.Name, suffix),
+		"component": "aerogear-app-metrics",
```

Comment:
@laurafitzgerald You should be able to use this new label to find the Route from your other controller, without having to know the CR name.